### PR TITLE
correct op spec to follow same optional output semantics.

### PR DESCRIPTION
**Pull request by linkerzhang**
This is a following-up PR of https://github.com/onnx/onnx/pull/1006 to clean all op specs with same 'optional" output semantics.

Will refer to the same piece of documentation once PR #1006 merged.

### CODE REVIEW

File: docs/Operators.md [hunk 1]
```diff
@@ -1161,13 +1159,13 @@ Other versions of this operator: <a href="Changelog.md#BatchNormalization-1">Bat
 <dt><tt>Y</tt> : T</dt>
 <dd>The output tensor of the same shape as X.</dd>
 <dt><tt>mean</tt> (optional) : T</dt>
-<dd>The running mean after the BatchNormalization operator. Must be in-place with the input mean. Should not be used for testing.</dd>
+<dd>The running mean after the BatchNormalization operator. Must be in-place with the input mean.</dd>
```
Comment (gramalingam) [inline docs/Operators.md]: Omit "Must be in-place with the input mean"

File: docs/Operators.md [hunk 2]
```diff
@@ -1161,13 +1159,13 @@ Other versions of this operator: <a href="Changelog.md#BatchNormalization-1">Bat
...
 <dt><tt>var</tt> (optional) : T</dt>
-<dd>The running variance after the BatchNormalization operator. Must be in-place with the input var. Should not be used for testing.</dd>
+<dd>The running variance after the BatchNormalization operator. Must be in-place with the input var.</dd>
```
Comment (gramalingam) [inline docs/Operators.md]: Omit reference to in-place update

File: docs/Operators.md (hunk 1 above)
Comment (linkerzhang) [reply docs/Operators.md]: will fix.

File: docs/Operators.md (hunk 2 above)
Comment (linkerzhang) [reply docs/Operators.md]: will fix.

**Pull request merged**